kijiji_repost_headless/kijiji_api.py

```diff
@@ -194,7 +194,7 @@ def post_ad_using_data(self, data, image_files=[]):
         new_ad_url = "https://www.kijiji.ca/p-submit-ad.html"
         resp = self.session.post(new_ad_url, data=data)
         resp.raise_for_status()
-        if "Delete Ad?" not in resp.text:
+        if "Delete Ad?" not in resp.text and "Supprimer l'annonce" not in resp.text:
```

Comment:
Should it be _or_, not _and_? Will the page have both english and french on it?

Reply:
In the english version, a successful ad post should say "Delete Ad?" in the text.

In the french version, a successful ad post should say "Supprimer l'annonce" in the text.

I think the logic is correct.

Reply:
Yes, you're right; if neither sentence is found on the page then the ad wasn't sucessful. Had to recall De Morgan's laws again.

Reply:
Why not using something agnostic from the language, like the markup? 

Like the presence of `id="ManageAdWidget"` or `class="edit-ad-begin"`.

Reply:
Is that method reliable? I'd agree that it would be better not to rely on printed text on the page.

Reply:
Equally reliable IMO, Kijiji could change the text at any moment, same goes for the markup. It's just better in the way that it is language agnostic. I'm guessing `resp.text` is actually the HTML return, so searching for a class/id should work.